### ISSUE TITLE
Logging in to services

### ISSUE THREAD

**Issue by forteller** [antimatter15/drag2up#8]
Many of the services has an option for logging in so you can collect all your files. It would be awesome if you could add support for logging in to those services who have an API that supports that. I'm pretty sure min.us would support that, for example.

**Comment by js365** [comment]
Yes, the log in ability for min.us is needed. It would be better to pop-up a box to let user choose to log in or share as a guest.
